plugins/admin-settings/general-settings/index.jsx

```diff
@@ -0,0 +1,161 @@
+import { __ } from '@wordpress/i18n';
+import { Button, TextControl, withNotices } from '@wordpress/components';
+import ImagePicker from '@/components/imagePicker';
+import { Sortable, SortableItem, useOption } from '@alleyinteractive/block-editor-tools';
+
+const AdminGeneralSettings = withNotices(
+  ({ noticeOperations, noticeUI }) => {
+    const {
+      value: settings = {},
+      isEdited = false,
+      isSaving = false,
+      onChange = () => {},
+      onSave = () => {},
+    } = useOption('nb_settings');
```

Comment:
I think we should make a custom hook for working with Newsletter Builder settings that does a few things:
1. It would wrap `useOption` so that we would eliminate duplicate code where we're extracting values from the settings object.
2. We could verify that the value returned from the call is of the proper shape and type so TypeScript is happy (and could convert this file to a TypeScript file while we're at it).
3. We could establish defaults when values don't exist.